formats/financial_releases_geoblocker/frontend/examples/financial_releases_not_permitted.json

```diff
@@ -1,6 +1,6 @@
 {
   "content_id": "054e8890-8cba-47fd-ab3b-32fc7a3ca2b3",
-  "base_path": "/financial-releases-geoblocker/not-permitted",
```

Comment:
I think we need to update this to `/lloydsshares/location/not-permitted` because it's an example format.